src/Adapter/ORM/DoctrineORMAdapter.php

```diff
@@ -14,19 +16,11 @@
 final class DoctrineORMAdapter implements AdapterInterface
 {
     /**
-     * @param \Doctrine\ORM\Event\LifecycleEventArgs $event
+     * @param LifecycleEventArgs $event
```

Comment:
I wonder if this could be replaced with `assert($event instanceof LifecycleEventArgs)` in the method body.